src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -8366,8 +8259,10 @@ Real64 CalcGoldsteinNovoselacCeilingDiffuserWindow(EnergyPlusData &state,
     if (ZoneExtPerimLength > 0.0) {
         if (WindWallRatio <= 0.5) {
 
-            if (WindowLocationType != InConvWinLoc_UpperPartOfExteriorWall && WindowLocationType != InConvWinLoc_LowerPartOfExteriorWall &&
-                WindowLocationType != InConvWinLoc_LargePartOfExteriorWall && WindowLocationType != InConvWinLoc_NotSet) {
+            if (WindowLocationType != ConvectionConstants::InConvWinLoc::UpperPartOfExteriorWall &&
```

Comment:
Can use `!BITF_TEST_ANY` if this enumeration is compact.

Reply:
OK, I'll take a look and convert over if possible.